azkaban-web-server/src/main/java/azkaban/imagemgmt/daos/ImageVersionDao.java

```diff
@@ -0,0 +1,11 @@
+package azkaban.imagemgmt.daos;
+
+import azkaban.imagemgmt.exeception.ImageMgmtException;
+import azkaban.imagemgmt.models.ImageVersion;
+import azkaban.imagemgmt.models.RequestContext;
+import java.util.List;
+
+public interface ImageVersionDao {
+  public int createImageVersion(ImageVersion imageVersion);
```

Comment:
Can you please add java-doc for all the public methods, interfaces and classes?

Reply:
Done.